Editor/UI/Drawers/TabsGroupDrawer.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using System.Linq;
+using Innoactive.Creator.Core.Tabs;
+using UnityEditor;
+using UnityEngine;
+
+namespace Innoactive.CreatorEditor.UI.Drawers
+{
+    [DefaultTrainingDrawer(typeof(ITabsGroup))]
+    internal class TabsGroupDrawer : AbstractDrawer
+    {
+        public override Rect Draw(Rect rect, object currentValue, Action<object> changeValueCallback, GUIContent label)
+        {
+            ITabsGroup tabsGroup = (ITabsGroup)currentValue;
+
+            GUIContent[] labels = tabsGroup.Tabs.Select(tab => tab.Label).ToArray();
+
+            int oldSelected = tabsGroup.Selected;
+
+            Rect nextPosition = new Rect(rect.x, rect.y, rect.width, EditorStyles.toolbar.fixedHeight);
+
+            int selected = GUI.Toolbar(nextPosition, oldSelected, labels);
+
+            if (selected != oldSelected)
+            {
+                ChangeValue(() =>
+                    {
+                        tabsGroup.Selected = selected;
+                        return tabsGroup;
+                    },
+                    () =>
+                    {
+                        tabsGroup.Selected = oldSelected;
+                        return tabsGroup;
+                    },
+                    changeValueCallback);
+            }
+
+            float height = EditorStyles.toolbar.fixedHeight;
+            nextPosition.position += new Vector2(0, height);
+            rect.height = EditorStyles.toolbar.fixedHeight
```

Comment:
Would love to see this not as an one liner, I have no clue what happens there :)